.github/workflows/ci.yml

```diff
@@ -5,8 +5,8 @@ jobs:
     name: Test
     strategy:
       matrix:
-        python-version: ['3.6.x', '3.7.x', '3.8.x']
-        django-version: ['2.1.15', '2.2.17', '3.0.11', '3.1.4']
+        python-version: ['3.8.x', '3.9.x', '3.10.x']
```

Comment:
better to test 3.9 to 3.11.. we'll be on 3.11 in a year

Reply:
Added 3.11 but just kept 3.8 as well for now